jdi-bdd-no-po-tests/src/test/java/cucumberTests/steps/ScenarioHooks.java

```diff
@@ -0,0 +1,14 @@
+package cucumberTests.steps;
+
+import cucumber.api.java.After;
+
+import static com.epam.jdi.light.elements.init.UIFactory.$;
+
+
+public class ScenarioHooks {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'ScenarioHooks' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5001515)